corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -144,7 +144,7 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
                         </div>
                         <div class="col-sm-2 controls">
                             <div class="form-check-placeholder">
-                                <label>&mdash;</label>
+                                <label></label>
```

Comment:
The empty label elements are weird to read; I'm not sure this commit is an improvement.

Reply:
the label is needed for the css styling.

Reply:
the dash comes from the border of label now rather than `&mdash;`

Reply:
Yeah, I didn't mean remove the label, just that reading the HTML makes more sense when the &mdash; is there.